core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow-list-item/user-workflow-list-item.component.ts

```diff
@@ -59,7 +60,8 @@ export class UserWorkflowListItemComponent {
   workflowExecutionsTrackingEnabled: boolean = environment.workflowExecutionsTrackingEnabled;
 
   constructor(
-    private modalService: NgbModal,
+    private modalService: NzModalService,
+    private ngbModalService: NgbModal,
```

Comment:
i thought you are replacing ngbModal with nzModal?

Reply:
The ngb modal left is for workflow Shay.